templates/components/tools/editors.hbs

```diff
@@ -19,6 +19,10 @@
     <a href="https://www.eclipse.org/downloads/packages/release/2019-09/r/eclipse-ide-rust-developers-includes-incubating-components"
        class="button button-secondary">{{fluent "tools-editor-eclipse"}}</a>
   </div>
+  <div class="w-25-l w-50-m w-100 pa3">
+    <a href="https://helix-editor.com"
```

Comment:
Most other editors have a link to some kind of documentation for Rust support specifically, rather than the generic landing-page for that editor. What do you think about linking to the rust-analyzer instructions instead (https://rust-analyzer.github.io/manual.html#helix)? They're very short, but the helix landing-page is also linked there and it makes it clear that rust-analyzer is not baked into the helix binary itself, but has to be installed separately.